store/store.go

```diff
@@ -0,0 +1,212 @@
+package store
+
+//import "log"
+import "time"
+import "golang.org/x/crypto/blake2b"
+import "sync"
+import "fmt"
+
+type NodeID [32]byte
```

Comment:
`node.ID` has been merged, so you can change all occurrences of `NodeID` to `node.ID`

Reply:
Sweet!